dashboard/modules/job/job_head.py

```diff
@@ -85,6 +85,16 @@ async def get_job_info(self, job_id: str) -> JobDetails:
             else:
                 await self._raise_error(resp)
 
+    async def get_job_logs(self, job_id: str) -> str:
+        async with self._session.get(
+            self._address + f"/api/job_agent/jobs/{job_id}/logs"
```

Comment:
The `_address` should be any of the agent right? What if the agent failed?

Also it took me a while to realize it's the agent address. Please rename it to agent_address.